invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html

```diff
@@ -9,54 +9,76 @@
 #}
 
 {% macro creatibutor_icon(creatibutor) %}
-{% set identifier_found = namespace(value=False) %}
-{% for scheme, identifier in creatibutor.person_or_org.identifiers|groupby("scheme") %}
-{%- if scheme == "orcid" %}
-{% set identifier_found.value = True %}
-<a href="{{ identifier[0]['identifier']|pid_url('orcid') }}" aria-label="{{ creatibutor.person_or_org.name }}'s ORCID Profile" title="{{ creatibutor.person_or_org.name }}'s ORCID Profile">
-  <img alt="" class="inline-id-icon" src="{{ url_for('static', filename='images/orcid.svg') }}" />
-</a>
-{%- elif scheme == "ror" %}
-{% set identifier_found.value = True %}
-<a href="{{ identifier[0]['identifier']|pid_url('ror') }}" aria-label="{{ creatibutor.person_or_org.name }}'s ROR Profile" title="{{ creatibutor.person_or_org.name }}'s ROR Profile">
-  <img alt=""  class="inline-id-icon" src="{{ url_for('static', filename='images/ror-icon.svg') }}" />
-</a>
-{%- endif %}
-{% endfor %}
-{# if no identifiers: distinguish btw people and organizations #}
-{%- if not identifier_found.value and creatibutor.person_or_org.type == 'organizational'%}
-<i class="group icon"></i>
-{%- endif %}
+  {% set identifier_found = namespace(value=False) %}
+
+  {% for scheme, identifier in creatibutor.person_or_org.identifiers|groupby("scheme") %}
+    {%- if scheme == "orcid" %}
+      {% set identifier_found.value = True %}
+      <a class="identifier-link" href="{{ identifier[0]['identifier']|pid_url('orcid') }}" aria-label="{{ creatibutor.person_or_org.name }}'s ORCID Profile" title="{{ creatibutor.person_or_org.name }}'s ORCID Profile">
+        <img class="inline-id-icon" src="{{ url_for('static', filename='images/orcid.svg') }}" alt="ORCID icon"/>
+      </a>
+    {%- elif scheme == "ror" %}
+      {% set identifier_found.value = True %}
+      <a href="{{ identifier[0]['identifier']|pid_url('ror') }}" aria-label="{{ creatibutor.person_or_org.name }}'s ROR Profile" title="{{ creatibutor.person_or_org.name }}'s ROR Profile">
+        <img class="inline-id-icon" src="{{ url_for('static', filename='images/ror-icon.svg') }}" alt="ROR icon"/>
+      </a>
+    {%- endif %}
+  {% endfor %}
+
+  {# if no identifiers: distinguish btw people and organizations #}
+  {%- if not identifier_found.value and creatibutor.person_or_org.type == 'organizational'%}
+    <i class="group icon"></i>
+  {%- endif %}
 {% endmacro %}
 
+
 {% macro show_creatibutors(creatibutors, show_affiliations=False) %}
   {% for creatibutor in creatibutors %}
+  <dd class="creatibutor-wrap">
     {{ creatibutor_icon(creatibutor) }}
-    <a
-      class="ui tooltip-popup text-muted"
+    <a class="ui tooltip-popup text-muted creatibutor-link"
       {% if show_affiliations and creatibutor.affiliations %}
         data-content="{{ creatibutor.affiliations|join('; ', attribute='1') }}"
```

Comment:
ROR icon should be added here as well I think:
![Screenshot 2022-02-03 at 15 07 48](https://user-images.githubusercontent.com/1698163/152362718-66ebbf75-2270-4e64-9ae9-35dcc3529fef.png)